web/static/css/index.css

```diff
@@ -62,4 +116,31 @@ a, a:hover, a:visited, a:link, a:active
 
     margin-top: 2rem;
     margin-bottom: 3rem;
+}
+
+.magicbox {
+    background-color: #2a2a2a;
+    height: 30rem;
+    margin-top: 5px;
+    margin-bottom: 5px;
+    width: 100%;
+    padding: 10px;
+    overflow: hidden;
+}
+
+.magicbox h3 {
+    margin-top: 0;
+    margin-bottom: 5px;
+}
+
+.col--small {
+    padding: 0 5px;
+}
+
+.table-striped>tbody>tr:nth-of-type(odd) {
+    background-color: #1f1f1f;
+}
+
+.table>tbody>tr>td {
+    border:none;
```

Comment:
I think we should start spliting the css in different sections like `bootstrap-theme.css` and `base.css` and `components.css` and `listUsers.css` etc.